src/main/kotlin/com/ampnet/mailservice/service/impl/MailServiceImpl.kt

```diff
@@ -94,66 +103,90 @@ class MailServiceImpl(
         )
         val tokenIssuersMessage = templateService.generateTextForTokenIssuerWithdrawRequest(userData)
         val tokenIssuersMail =
-            createMailMessage(tokenIssuers.map { it.email }, manageWithdrawalsSubject, tokenIssuersMessage)
+            createMailMessages(tokenIssuers.map { it.email }, manageWithdrawalsSubject, tokenIssuersMessage)
 
         val userMessage = templateService.generateTextForWithdrawRequest(
             AmountData(TWO_DECIMAL_FORMAT.format(amount / FROM_CENTS_TO_EUROS))
         )
-        val userMail = createMailMessage(listOf(user.email), withdrawSubject, userMessage)
+        val userMail = createMailMessages(listOf(user.email), withdrawSubject, userMessage)
 
-        sendEmail(tokenIssuersMail)
-        sendEmail(userMail)
+        sendEmails(tokenIssuersMail)
+        sendEmails(userMail)
     }
 
     override fun sendWithdrawInfoMail(user: UserResponse, burned: Boolean) {
         val data = WithdrawInfo(burned)
         val message = templateService.generateTextForWithdrawInfo(data)
-        val mail = createMailMessage(listOf(user.email), withdrawSubject, message)
-        sendEmail(mail)
+        val mail = createMailMessages(listOf(user.email), withdrawSubject, message)
+        sendEmails(mail)
     }
 
     override fun sendNewWalletNotificationMail(walletType: WalletType) {
         val link = linkResolver.getNewWalletLink(walletType)
         val message = templateService.generateTextForNewWallet(NewWalletData(link), walletType)
         val platformManagers = userService.getPlatformManagers()
-        val mail = createMailMessage(platformManagers.map { it.email }, newWalletSubject, message)
-        sendEmail(mail)
+        val mail = createMailMessages(platformManagers.map { it.email }, newWalletSubject, message)
+        sendEmails(mail)
     }
 
     override fun sendWalletActivatedMail(walletOwner: String, walletType: WalletType) {
         val (walletActivatedData, userUUid) = getDataAndUser(walletOwner, walletType)
         val message = templateService.generateTextForWalletActivated(walletActivatedData, walletType)
         val userEmail = userService.getUsers(listOf(userUUid)).map { it.email }
-        val mail = createMailMessage(userEmail, walletActivatedSubject, message)
-        sendEmail(mail)
+        val mail = createMailMessages(userEmail, walletActivatedSubject, message)
```

Comment:
Variable is in singular but function returns list(plural). Change the name or function.